adapters/coap-vertx-base/src/main/java/org/eclipse/hono/adapter/coap/CoapContext.java

```diff
@@ -45,14 +45,20 @@
 
     private final CoapExchange exchange;
     private final Device originDevice;
+    private final String authId;
```

Comment:
isn't this useful for all other adapters as well, i.e. doesn't it make sense to pull it up into MapBasedTelemetryExecutionContext?

Reply:
Currently that doesn't look useful. But it might be when doing the refactorings concerning the *TenantObjectWithIdProvider classes.